Tensile/Common.py

```diff
@@ -1373,6 +1354,16 @@ def getArchitectureName(gfxName):
     # 0=none, 1=add setprio, 2=add setprio and modify LDS to allow only 2 waves/simd
     "AggressivePerfMode":       [0,1,2],
 
+    # Use the feature whereby 56 bytes of kernel arguments can be preloaded in SGPRs
+    # before the kernel begins executing.  This is currently only supported in cases
+    # where that is enough to initiate the first load of the A and B tensors before
+    # the remaining arguments have been loaded.
+    "PreloadKernelArguments": [False, True],
```

Comment:
I thought you would define the default values for these 2 new parameters.
Any benefit of not defining default valu?

Reply:
Yes, this way it will default to on if supported, off if not.  We discussed changing the default to -1 or similar if it was causing problems, but it was not the cause of the problems.

I can still make it that way if needed.